stockflux-securities-master/package.json

```diff
@@ -5,13 +5,20 @@
   "dependencies": {
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "3.0.1"
+    "react-scripts": "3.0.1",
+    "openfin-fdc3": "^0.1.1",
+    "openfin-layouts": "^1.0.0",
+    "openfin-react-hooks": "^1.0.1",
+    "stockflux-core": "file:../stockflux-core",
+    "stockflux-components": "file:../stockflux-components"
   },
+  "homepage": "./",
```

Comment:
I think this needs to be a URL to a website. Maybe it should point to the GitHub repo for now (or the stockflux.scottlogic.com domain which is coming soon), or maybe it should be deleted since it's not needed?